libs/fundamental-redirects/index.js

```diff
@@ -1168,7 +1199,8 @@ const REDIRECT_PATTERNS = [].concat(
       { prependLocale: false, permanent: true }
     ),
   ],
-  LOCALE_PATTERNS
+  LOCALE_PATTERNS,
+  SIMPLE_REDIRECT_PATTERNS
```

Comment:
This also conflicts with `SCL3_REDIRECT_PATTERNS`:
```js
  redirect(
    /^En\/Core_JavaScript_1\.5_Reference\/Objects\/(?<suffix>.*)$/i,
    ({ suffix }) => `/en-US/docs/JavaScript/Reference/Global_Objects/${suffix}`,
    { permanent: true }
  ),
  // RewriteRule ^En/Core_JavaScript_1\.5_Reference/(.*)
  // en-US/docs/JavaScript/Reference/$1 [R=301,L,NC]
  redirect(
    /^En\/Core_JavaScript_1\.5_Reference\/(?<suffix>.*)$/i,
    ({ suffix }) => `/en-US/docs/JavaScript/Reference/${suffix}`,
    { permanent: true }
  ),
```

Reply:
So can we delete these?

Reply:
I thought we already had a thing that converts any `/En/...` to `/en-US/docs/...` no?